cmd/bootstrap/PROVIDED_CERTIFICATES.md

```diff
@@ -0,0 +1,150 @@
+# User provided certificates
+
+Users can provide their own certificates to be used with Skupper V2 in non kube sites
+during the bootstrap of a local site, when preparing a site bundle and even while installing
+a site bundle at a remote machine. 
+
+## How certificates are used internally
+
+During the initialization of a local non kube site or when a site bundle is being prepared,
+Skupper generates an internal Certificate Authority (CA) named `skupper-site-ca`.
+
+If a `RouterAccess` is defined as part of the non kube site definition, Skupper will also
+generate a server certificate that is valid for the provided `RouterAccess.spec.bindHost` and
+for each entry in the  `RouterAccess.spec.subjectAlternativeName` list. The respective certificate
+is signed by the CA mentioned earlier (skupper-site-ca).
+
+A set of static links are also created for the provided `RouterAccess.spec.bindHost` and for each
+entry in the  `RouterAccess.spec.subjectAlternativeName` list as separate YAML files.
+
+These static links are composed by a `Link` and a `Secret`, which is basically the client certificate
+signed by `skupper-site-ca`, that will be used by other sites to establish Skupper links.
+
+After a site bundle has been produced, it contains the whole site definition that can be installed
+at a remote location. Thus, all certificates and static links are already part of the bundle and no
+new certificate is signed at the moment a bundle is installed.
+
+## Providing your own certificates
+
+You can provide your own certificates to be used by Skupper for site linking.
+Depending on your goals, some certificates should be supplied at certain phases,
+for example, at the time a local site is being initialized, a bundle is being prepared
+or a bundle is being installed at a remote location.
+
+To understand it better, let's go through the main use cases and review what is the ideal
+phase that a given kind of certificate should be provided.
+
+### Using a custom CA to sign certificates
+
+If you want Skupper to generate client and server certificates signed by a custom CA,
+you will need to provide the respective certificates during:
```

Comment:
I presume the custom CA needs to use a specific filename? Or are they provided in the form of a kubernetes secret with a given object name?